lib/resources/r4/questionnaire_response.yaml

```diff
@@ -23,4 +36,214 @@ fields:
+                  }​​​​​​​​​​​​​​​​​​​​​​​​​​
+                ],
+                "answer": [
+                  {
+                    "valuestring": "Answer text"
+                  }
+                ]
+              }
+            ] 
+          }
+        ]
+      }
+
+  - name: item.id
+    required: 'No'
+    type: string
+    url: https://hl7.org/fhir/R4/element-definitions.html#Element.id
+    description: Unique id for inter-element referencing.
+    action: update
+    example: |
+      {
+        "id": "111"
+      }
+    note: If an item.id is returned on a read, it must be provided on an update.
+
+  - name: item.linkId
+    required: 'Yes'
+    type: string
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.linkId
+    description: The item from the Questionnaire that corresponds to this item in the QuestionnaireResponse resource.
+    action: update
+    example: |
+      {
+        "linkId": "93"
+      }
+
+  - name: item.extension
+    required: 'No'
+    type: Extension
+    url: https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/StructureDefinition/note
+    description: Additional details about a given QuestionnaireResponse group item.
+    action: update
+    example: |
+      {
+        "extension": [
+          {​​​​​​​​​​​​
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/note",
+            "valueAnnotation": {​​​​​​​​​​​​​​​​​​​
+              "text": "category/group comment"
+            }​​​​​​​​​​​​​​​​​​​​​​​​​​
+          }​​​​​​​​​​​​​​​​​​​​​​​​​​
+        ]
+      }
+    note: Notes can only be added, not replaced or deleted.
+
+  - name: item.item
+    required: 'No'
+    type: List of BackboneElement
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.item
+    description: Questions nested beneath a group.
+    action: update
+    example: |
+      {
+        "item": [
+          {
+            "linkId": "93-123",
+            "extension": [
+              {​​​​​​​​​​​​
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/choice-answer",
+                "valueBoolean": true
+              }​​​​​​​​​​​​​​​​​​​​​​​​​​
+            ],
+            "answer": [
+              {
+                "valuestring": "Answer text"
+              }
+            ]
+          }
+        ]
+      }
+
+  - name: item.item.linkId
+    required: 'Yes'
+    type: string
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.linkId
+    description: The item from the Questionnaire that corresponds to this item in the QuestionnaireResponse resource.
+    action: update
+    example: |
+      {
+        "linkId": "93-123"
+      }
+
+  - name: item.item.extension
+    required: 'No'
+    type: Extension
+    url: https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/StructureDefinition/choice-answer
```

Comment:
i think the url is just https://fhir-ehr.cerner.com/r4/StructureDefinition/choice-answer actually, same for note extension

Reply:
ac56d6a6e28c12bcc72cc61c5a6aaca88380c29e